packages/ui/src/components/molecules/Aside/InfoAside.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement } from 'react';
+import { styled } from 'styled-components';
+import { theme } from '../../../themes/theme.styled';
+import {
+  AsideContainer,
+  Flex,
+  Image,
+  LangDisplay,
+  Typography,
+} from '../../atoms';
+import { cysyncLogoBig, cysyncLogoSmall } from '../../../assets/images';
+
+export interface InfoAsideProps {
+  title: string;
+  subTitle: string;
+  version: string;
+}
+
+const HeadingFiveText = styled.span`
```

Comment:
Maybe we can change to this?

```suggestion
const HeadingFiveText = styled.h5`
```